src/wp-includes/revision.php

```diff
@@ -463,6 +513,101 @@ function wp_restore_post_revision( $revision, $fields = null ) {
 	return $post_id;
 }
 
+/**
+ * Restore the revisioned meta values for a post.
+ *
+ * @param int $post_id     The ID of the post to restore the meta to.
+ * @param int $revision_id The ID of the revision to restore the meta from.
+ *
+ * @since 6.4.0
+ */
+function wp_restore_post_revision_meta( $post_id, $revision_id ) {
+
+	// Restore revisioned meta fields.
+	foreach ( wp_post_revision_meta_keys( get_post_type( $post_id ) ) as $meta_key ) {
```

Comment:
https://github.com/WordPress/wordpress-develop/pull/4859/files#r1335858255

Reply:
see above, maybe fine to pass false here?